workflow-service/src/test/java/com/redhat/parodos/workflow/definition/service/WorkFlowDefinitionServiceDataTest.java

```diff
@@ -69,10 +90,10 @@ void cleanAllDefinition_should_returnEmpty_when_DatabaseIsCleaned() {
 		assertThat(workFlowCheckerMappingDefinitionRepository.findAll()).hasSize(1);
 		assertThat(workFlowWorkRepository.findAll()).hasSize(1);
 
-		workFlowDefinitionService.cleanAllDefinitions();
+		workFlowDefinitionService.cleanAllDefinitionMappings();
```

Comment:
The method name is `cleanAllDefinition_should_returnEmpty_when_DatabaseIsCleaned` can you update it?
I mean before this change, all the methods were supposed to be empty now only `workFlowCheckerMappingDefinitionRepository` and `workFlowWorkRepository` are empty.